src/configuration.ts

```diff
@@ -26,7 +26,7 @@ export class ConnectionConfiguration implements IConnectionConfiguration {
     this.debugFilePath = this.config['languageserver']['debugFilePath'];
   }
 
-  get puppetAgentDir(): string {
+  get puppetBaseDir(): string {
```

Comment:
I didn't have to change the name of the accessor here, but it helped keep things straight in my head.